File: jsshin8128/3진법뒤집기.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

int solution(int n) {
    int answer = 0;
    vector<int> v;

    while (n > 0) {
        v.push_back(n % 3);
        n /= 3;
    }

    for (int i = v.size() - 1; i >= 0; i--) {
        answer += v[v.size() - i - 1] * int(pow(3, i));
    }

    return answer;
}

File: reewon/Week3/3진법 뒤집기.py
def radixChange(num, radix):
    if num == 0: return 0
    nums = []
    while num:
        num, digit = divmod(num, radix)
        nums.append(str(digit))
    return "".join(nums)
    
def solution(n):
    n = radixChange(n, 3)
    answer = int(str(n), 3)
    return answer
